actions.yml

```diff
@@ -101,6 +101,7 @@ imjohnbo/issue-bot:
     issues: write
     repository-projects: write
 cypress-io/github-action:
+8398a7/action-slack:
```

Comment:
```
cypress-io/github-action:
  permissions:
    actions: read
8398a7/action-slack:
  permissions:
    actions: read
```
It would be correct.

Reply:
```suggestion
  permissions:
    actions: read
8398a7/action-slack:
```